microcosm_flask/swagger/definitions.py

```diff
@@ -120,21 +121,53 @@ def iter_definitions(definitions, operations):
         yield get_response_schema(func)
 
 
-def build_path(operation, ns):
+def build_path_for_integer_param(ns, operation, arguments):
     """
-    Build a path URI for an operation.
+    Build an endpoint path when the parameters are integer-valued
+
+    When building paths for swagger, parameter names are substituted for
+    path parameters. For example, the output will have '/api/v1/person/{person_id}'.
+    We still use werkzeug to build those paths with placeholders for params.
+
+    That works with UUID-valued ids but not integer-valued ones, due do a difference
+    between the UUIDConverter and NumberConverter's `to_url` methods
+    (see https://github.com/pallets/werkzeug/blob/master/src/werkzeug/routing.py#L1315
+    and https://github.com/pallets/werkzeug/blob/master/src/werkzeug/routing.py#L1234)
+
+    Here we instead use placeholder integers to use werkzeug functions, and replace
+    them afterwards.
 
     """
+    sorted_args = list(arguments)
```

Comment:
How does casting them to a list making them sorted? are you missing a `sorted` or is the input to this function expecting this to be pre-sorted?

Reply:
`arguments` is a set. `ordered_args` would be a better name than `sorted_args`.

Reply:
Do we using `typing` in this library? If so might be nice to add the type definitions to help